sys-block/open-iscsi/open-iscsi-2.1.10-r1.ebuild

```diff
@@ -89,8 +88,9 @@ src_configure() {
 src_install() {
 	meson_src_install
 
-	# Upstream make is not deterministic, per bug #601514
-	rm -f "${ED}"/etc/initiatorname.iscsi
+	# We'll regenerate this later to avoid baking the value into binary
+	# packages. It doesn't get generated when cross-compiling.
+	rm -f "${ED}"/etc/iscsi/initiatorname.iscsi
```

Comment:
This shows that `-f` was wrong here. Add instead a check for `tc-is-cross-compiler` and if it's false, rm the file & die with no -f.

Reply:
Okay, will do that and merge.